no-downtime-upgrade/src/main/java/org/keycloak/playground/nodowntimeupgrade/infinispan/v1/ObjectModelV1DelegateV1.java

```diff
@@ -0,0 +1,47 @@
+package org.keycloak.playground.nodowntimeupgrade.infinispan.v1;
+
+import org.keycloak.playground.nodowntimeupgrade.base.model.ObjectModel_V1;
+
+public class ObjectModelV1DelegateV1 implements ObjectModel_V1 {
```

Comment:
We don't want to delegate on the level of models but on the level of entities, since no-downtime upgrade is expected to work with MapStorage low-level CRUD operations.